llama_stack/providers/datatypes.py

```diff
@@ -172,10 +179,12 @@ def from_url(cls, url: str) -> "RemoteProviderConfig":
 
 @json_schema_type
 class RemoteProviderSpec(ProviderSpec):
-    adapter: AdapterSpec = Field(
+    adapter: Optional[AdapterSpec] = Field(
```

Comment:
this seems to have come back from the dead

Reply:
done.